packages/target-decisioning-engine/src/contextProvider.js

```diff
@@ -27,6 +28,22 @@ function getLowerCaseAttributes(obj) {
   return result;
 }
 
+function withLowerCaseStringValues(obj) {
```

Comment:
big fan of the recursion - makes more sense for the data structure and fixes the _lc bug